### DISCUSSION TITLE
Menu link to form with steps

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#7585] [Help] [answered]
In my navigation links, I have a link to a form page that has several steps.
If I am on step 3 for example, and I click the link in the navigation bar, I'd like to go back to step 1.

But it doesn't do anything, it remains on step 3.

Is there a way to make it "reload" the page so that it goes to step 1 ?

**Comment by melohagan** [comment, marked as answer]
Hi @NicolasFabre81 

You may need to add a custom "Restart form" button / "Go to start" button. 

Whenever you click on a navigation link and you are already on that page then it doesn't refresh by design. 

Another possibility could be to have the first form step on a different screen. 
 - Navigation link to Step 1 screen
 - Second screen with the form steps (no navigation link)

In this case clicking a 'Next' button in the Step 1 screen would navigate you to the second screen. Then the navigation link for the Step 1 screen would be available.
You may need to store the form state of step 1 with app state: https://docs.budibase.com/docs/app-state
